apps/web/src/pages/User/index.tsx

```diff
@@ -104,13 +104,13 @@ const User = () => {
           />
         )}
         <StyledTitle>{user?.name}</StyledTitle>
+        <FollowButton onClick={handleFollowUser}>
+          {isFollowingData?.user?.isFollowing ? 'Unfollow' : 'Follow'}
+        </FollowButton>
+        <h4>{isFollowingData?.user?.followerCount} Followers</h4>
       </StyledUserContainer>
       {user ? (
         <StyledProjectContainer>
-          <FollowButton onClick={handleFollowUser}>
```

Comment:
why was this moved?

Reply:
seems like a bad placement for the button UI wise, which was on top of the project Grid seems resonable to be grouped together with the user avatar pic, name an followers count